web/src/components/molecules/Integration/IntegrationTable/index.tsx

```diff
@@ -0,0 +1,100 @@
+import styled from "@emotion/styled";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import ConfigProvider from "@reearth-cms/components/atoms/ConfigProvider";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import ProTable, { ListToolBarProps } from "@reearth-cms/components/atoms/ProTable";
+import { useT } from "@reearth-cms/i18n";
+
+const columns = [
+  {
+    title: "Name",
+    dataIndex: "name",
+    key: "name",
+    filters: [],
+  },
+  {
+    title: "Role",
+    dataIndex: "role",
+    key: "role",
+  },
+  {
+    title: "Creator",
+    dataIndex: "creator",
+    key: "creator",
+  },
+  {
+    title: "State",
+    dataIndex: "state",
+    key: "state",
+  },
+  {
+    title: "Action",
+    dataIndex: "action",
+    key: "action",
+  },
+];
```

Comment:
Titles should use i18n. Please move the columns array in to the component and wrap them in t()

Reply:
@KaWaite , done.
https://github.com/reearth/reearth-cms/blob/f3350050914c6b800b371d5dd40f763c1ff7804c/web/src/components/molecules/Integration/IntegrationTable/index.tsx#L13-L40